comp/otelcol/otlp/components/exporter/serializerexporter/serializer.go

```diff
@@ -16,6 +17,7 @@ import (
 	metricscompression "github.com/DataDog/datadog-agent/comp/serializer/metricscompression/def"
 	metricscompressionfx "github.com/DataDog/datadog-agent/comp/serializer/metricscompression/fx-otel"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/datadog"
+	"golang.org/x/net/http/httpproxy"
```

Comment:
nit: could you reorganize the imports in this file? We typically organize imports in the following order but this file is a bit out of order:

```
import (
  stdlib (i.e. context, fmt, strings)

  datadog-agent deps (github.com/DataDog/datadog-agent/...)

  external deps (i.e. golang.org/x/net/http/httpproxy, github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes/source, github.com/open-telemetry/opentelemetry-collector-contrib/pkg/datadog)
)